midterm-2023-11-06/code/5265-genci-telqiu/script.ts

```diff
@@ -0,0 +1,121 @@
+type Band = {
```

Comment:
The types are correct (I personally prefer interfaces for this kind of data, but this is totally ok) +10